src/antsibull_core/__init__.py

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
```

Comment:
[The default encoding for Python 3 is already `utf-8`](https://peps.python.org/pep-3120/). I don't think this is necessary.

Reply:
/me pushed the wrong button again...

Reply:
That would have applied to antsibull as well :-)

Reply:
Implemented in b937bee5facc0faf680bba83792e536b26fa9b40.